Microsoft.Toolkit.Win32/Microsoft.Toolkit.Win32.UI.Controls/Microsoft.Toolkit.Win32.UI.Controls.csproj

```diff
@@ -22,15 +22,41 @@
       - /.vsts-pr.yml
 
     This also needs to be installed on your local machine. Can do this with PowerShell:
-      ./build/Install-WindowsSDKISO.ps1 17704
+      ./build/Install-WindowsSDKISO.ps1 17709
     -->
-    <TargetPlatformVersion>10.0.17704.0</TargetPlatformVersion>
+    <TargetPlatformVersion>10.0.17709.0</TargetPlatformVersion>
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
     <DefineConstants>TRACE;DEBUG;CODE_ANALYSIS;NET47;DEV_DEBUG;DEBUG_LAYOUT</DefineConstants>
   </PropertyGroup>
 
+  <ItemGroup>
+    <Compile Remove="WPF\InkCanvas\CoreInputDeviceTypes.cs" />
```

Comment:
❔

Reply:
This is confusion over the use of WinMD on the client side.  If the client is encouraged to use WinMD, these are all classes we don't need.  If they are discouraged from using WinMD, we need them.  For the time being, I pulled them out, but still wanted to have the issue reviewed.

Reply:
They are discouraged from WinMD. In the case for WebView, the developer could continue to do work on down-level OS / server where Edge doesn't exist (they could compile, but not actually run the web view).